docs/src/pages/docs/crocks/Arrow.md

```diff
@@ -349,7 +349,7 @@ Arrow a b ~> ((c -> a), (b -> d)) -> Arrow c d
 
 `promap` can be used to adapt BOTH ends of an `Arrow` allowing for existing
 `Arrow`s to be reused in places in a flow where the types do not line up. It
-combines both `map` and `conramap` into one operation. Just pass the `contramap`
+combines both `map` and `contramap` into one operation. Just pass the `contramap`
```

Comment:
The doucmentation [here](https://github.com/evilsoft/crocks/blob/master/docs/src/pages/docs/crocks/Arrow.md#promap) should also be updated to reflect the change.